src/app/lib/utilities/getToggles/index.js

```diff
@@ -12,6 +12,27 @@ import getOriginContext from '#contexts/RequestContext/getOriginContext';
 const logger = nodeLogger(__filename);
 const NS_PER_SEC = 1e9;
 
+const logResponseTime = async (url, origin, service, timeout) => {
+  const isBrowser =
```

Comment:
We have a utility class called `onClient` in `src/app/lib/utilities/onClient` which appears to do a similar check, just thinking it could be used here too?

Reply:
Sweet, I'll update it to use that instead.

Reply:
Sticking with the current implementation as I wasn't able to mock/spy on `onClient` properly.